test/unit/lib/classes/plugin-manager.test.js

```diff
@@ -586,9 +585,9 @@ describe('PluginManager', () => {
       const plugin1 = new ServicePluginMock1();
       plugin1.asyncInit = sinon.stub().returns(Promise.resolve());
       pluginManager.plugins = [plugin1];
-      return pluginManager.asyncPluginInit().then(() => {
-        expect(plugin1.asyncInit.calledOnce).to.equal(true);
-      });
+
+      await pluginManager.asyncPluginInit();
+      expect(plugin1.asyncInit.calledOnce).to.be.true;
```

Comment:
Let's not introce such stylistic changes (`to.equal(true)` into `to.be.true`), it brings on extra value, while increases maintenance cost